### ISSUE TITLE
Parser Issue

### ISSUE THREAD

**Issue by rudde0** [SkriptLang/Skript#4732]
### Skript/Server Version

```markdown
[21:15:57 INFO]: [Skript] Server Version: git-Purpur-1581 (MC: 1.18.2)
[21:15:57 INFO]: [Skript] Skript Version: 2.6.1
[21:15:57 INFO]: [Skript] Installed Skript Addons:
[21:15:57 INFO]: [Skript]  - skript-gui v1.3-alpha3-nightly-30455aa (https://github.com/APickledWalrus/skript-gui)
[21:15:57 INFO]: [Skript]  - skript-npc v0.1.2 (https://github.com/NanoDankster)
[21:15:57 INFO]: [Skript]  - skript-yaml v1.4.1
[21:15:57 INFO]: [Skript]  - skRayFall v1.9.26 (https://sk.rayfall.net/)
[21:15:57 INFO]: [Skript]  - SkBee v1.15.1 (https://github.com/ShaneBeee/SkBee)
[21:15:57 INFO]: [Skript]  - skript-advancements v1.0
[21:15:57 INFO]: [Skript]  - skript-placeholders v1.5.2 (https://github.com/APickledWalrus/skript-placeholders)
[21:15:57 INFO]: [Skript]  - skDragon v0.17
[21:15:57 INFO]: [Skript]  - skript-reflect v2.3 (https://github.com/TPGamesNL/skript-reflect)
[21:15:57 INFO]: [Skript]  - skript-db v1.3.4
[21:15:57 INFO]: [Skript]  - Skungee v2.0.0-BETA-2
[21:15:57 INFO]: [Skript]  - SkQuery v4.1.5
[21:15:57 INFO]: [Skript]  - skript-holo v1.0.2 (https://github.com/Blueyescat/skript-holo)
[21:15:57 INFO]: [Skript] Installed dependencies:
[21:15:57 INFO]: [Skript]  - Vault v1.7.3-b131
[21:15:57 INFO]: [Skript]  - WorldGuard v7.0.7+216b061
[21:15:57 INFO]: [Skript]  - GriefPrevention v${git.commit.id.describe}
```

### Bug Description

When I upgrade Skript 2.6.1 to latest nightly build I experienced an issue which resulting server crash.

### Expected Behavior

Elements should added to the list successfully.

### Steps to Reproduce

Execute effect command code below (or write your skript file)
`
add (32 of cornflower, 16 of cornflower, 64 of oxeye daisy, 4 of wither rose, 32 of rose bush, 8 of poppy, 32 of dandelion, 16 of brown mushroom, 2 of crimson fungus, 16 of allium and 8 of white tulip) to {ilkbaharRng::*} if {ilkbaharRng::*} isn't set
`

Please note that the code below working,
```
	set {ilkbaharRng::1} to 32 of cornflower
	set {ilkbaharRng::2} to 16 of cornflower
	set {ilkbaharRng::3} to 64 of oxeye daisy
	set {ilkbaharRng::4} to 4 of wither rose
	set {ilkbaharRng::5} to 32 of rose bush
	set {ilkbaharRng::6} to 8 of poppy
	set {ilkbaharRng::7} to 32 of dandelion
	set {ilkbaharRng::8} to 16 of brown mushroom
	set {ilkbaharRng::9} to 2 of crimson fungus
	set {ilkbaharRng::10} to 16 of allium
	set {ilkbaharRng::11} to 8 of white tulip
```

### Errors or Screenshots

```
[21:08:13] [Watchdog Thread/ERROR]: The server has not responded for 10 seconds! Creating thread dump
[21:08:13] [Watchdog Thread/ERROR]: ------------------------------
[21:08:13] [Watchdog Thread/ERROR]: Server thread dump (Look for plugins here before reporting to Purpur!):
[21:08:13] [Watchdog Thread/ERROR]: ------------------------------
[21:08:13] [Watchdog Thread/ERROR]: Current Thread: Server thread
[21:08:13] [Watchdog Thread/ERROR]: 	PID: 39 | Suspended: false | Native: false | State: RUNNABLE
[21:08:13] [Watchdog Thread/ERROR]: 	Stack:
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:516)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:741)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:126)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:516)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:741)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:126)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:516)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:741)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:126)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:516)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:741)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:126)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:516)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:741)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:126)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:516)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:741)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:126)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:516)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:741)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:126)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:516)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:741)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:126)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:516)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:741)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:126)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:121)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:516)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseSingleExpr(SkriptParser.java:399)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parseExpression(SkriptParser.java:741)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.TypePatternElement.match(TypePatternElement.java:126)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.PatternElement.matchNext(PatternElement.java:54)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.LiteralPatternElement.match(LiteralPatternElement.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.ChoicePatternElement.match(ChoicePatternElement.java:63)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.GroupPatternElement.match(GroupPatternElement.java:47)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.patterns.SkriptPattern.match(SkriptPattern.java:57)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1239)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:223)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.Effect.parse(Effect.java:76)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.effects.EffDoIf.init(EffDoIf.java:58)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:254)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:176)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.lang.Effect.parse(Effect.java:76)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.command.Commands.handleEffectCommand(Commands.java:273)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.command.Commands$3$1.call(Commands.java:217)
[21:08:13] [Watchdog Thread/ERROR]: 		Skript.jar//ch.njol.skript.command.Commands$3$1.call(Commands.java:214)
[21:08:13] [Watchdog Thread/ERROR]: 		org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftFuture.run(CraftFuture.java:88)
[21:08:13] [Watchdog Thread/ERROR]: 		org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483)
[21:08:13] [Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1562)
[21:08:13] [Watchdog Thread/ERROR]: 		net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:522)
[21:08:13] [Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1485)
[21:08:13] [Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1248)
[21:08:13] [Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:322)
[21:08:13] [Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer$$Lambda$4145/0x00000008014b7630.run(Unknown Source)
[21:08:13] [Watchdog Thread/ERROR]: 		java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
```

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by AyhamAl-Ali** [comment]
Duplicate of #1117
Skript finds it hard to parse long lines, it's a known minor issue.

**Comment by rudde0** [comment]
> Duplicate of #1117
> 
> Skript finds it hard to parse long lines, it's a known minor issue. Also, side note: use `add 32 of cornflower to {ilkbaharRng::*}` instead of setting each index

Are you sure to are they same issue? Because this problem cause to server crash.

**Comment by AyhamAl-Ali** [comment]
Positive, the crash log is about SkriptParser and the server hanged, it took too long parsing

**Comment by rudde0** [comment]
> Positive, the crash log is about SkriptParser and the server hanged, it took too long parsing
> 
> And since separating them works, it means the code is okay.

idk, I've waited like 20 mins when I add this code to skript file and I shutdown the server.

**Comment by AyhamAl-Ali** [comment]
Yea, don't use long line codes, just split them

**Comment by TPGamesNL** [comment]
Not a duplicate of #1117, but of #4025

#1117 is when there's a StackOverflowError with regex parsing in it, caused by node comment splitting of long lines

**Comment by rudde0** [comment]
> Not a duplicate of #1117, but of #4025
> 
> #1117 is when there's a StackOverflowError with regex parsing in it, caused by node comment splitting of long lines

This issue don't exist on Skript 2.6 btw
